gtests/net/mptcp/unsupported/test_server_1.pkt

```diff
@@ -0,0 +1,15 @@
+
+// connection initiated by packetdrill
+--tolerance_usecs=100000
+`../../common/defaults.sh`
+
+
++0     socket(..., SOCK_STREAM, IPPROTO_MPTCP) = 3
++0     setsockopt(3, SOL_SOCKET, SO_ZEROCOPY, [1], 4) = 0
```

Comment:
I guess this is unrelated to IP_TOS and this file should not be in this commit, right?